tests/unit/lms/resources/_js_config_test.py

```diff
@@ -199,12 +199,14 @@ def method(self, js_config, request):
 
 
 class TestMaybeEnableGrading:
-    def test_it_adds_the_grading_settings(self, js_config, grading_info_service):
+    def test_it_adds_the_grading_settings(
+        self, js_config, grading_info_service, pyramid_request
+    ):
         js_config.maybe_enable_grading()
 
         grading_info_service.get_by_assignment.assert_called_once_with(
             context_id="test_course_id",
-            oauth_consumer_key="TEST_OAUTH_CONSUMER_KEY",
+            oauth_consumer_key=pyramid_request.lti_user.oauth_consumer_key,
```

Comment:
Tests can no longer rely on `"TEST_OAUTH_CONSUMER_KEY"` having been hard-coded when `pyramid_request.lti_user` was created. In most cases accessing `pyramid_request.lti_user.oauth_consumer_key` instead of the literal value is the best solution.

I am starting to want to rename `pyramid_request` to `req` though